kubernetes/EKS/README.md

```diff
@@ -361,88 +400,49 @@
   Succesfully created EFS & Mountpoints
   ```
 
-  
-
   Upon completion of the script it would emit a EFS volume DNS Name similar to `fs-ab1cd.efs.us-west-2.amazonaws.com` where `fs-ab1cd` is the EFS filesystem id.
 
-  
-
   You should be able to see a Security Group in your AWS Console with Inbound Rules to a NFS (Port 2049)
 
-  
-
   ![security_group](images/security_group.png)
 
-  
-
   You should also find Mount Points in your EFS console for every region where there is a Node in the Node Group.
 
-  
 
   ![](images/efs_mount.png)
 
-  
-
-  
-
-  **Prepare PersistentVolume for Deployment**
-
-  We use the [ELF Provisioner Helm Chart](https://github.com/helm/charts/tree/master/stable/efs-provisioner) to create a PersistentVolume backed by EFS. Run the following command to set this up.
-
-  ```bash
-  helm repo add stable https://charts.helm.sh/stable
-  helm repo update
-  helm install stable/efs-provisioner --set efsProvisioner.efsFileSystemId=YOUR-EFS-FS-ID --set efsProvisioner.awsRegion=us-west-2 --set efsProvisioner.reclaimPolicy=Retain --generate-name
-  ```
+  Create a storage class file with below specs.
 
-  
-
-  you should get an output similar to 
-
-  
-
-  ```bash
-  NAME: efs-provisioner-1596010253
-  LAST DEPLOYED: Wed Jul 29 08:10:56 2020
-  NAMESPACE: default
-  STATUS: deployed
-  REVISION: 1
-  TEST SUITE: None
-  NOTES:
-  You can provision an EFS-backed persistent volume with a persistent volume claim like below:
-  
-  kind: PersistentVolumeClaim
-  apiVersion: v1
+  ```yaml
+  kind: StorageClass
+  apiVersion: storage.k8s.io/v1
   metadata:
-    name: my-efs-vol-1
-    annotations:
-      volume.beta.kubernetes.io/storage-class: aws-efs
-  spec:
-    storageClassName: aws-efs
-    accessModes:
-      - ReadWriteMany
-    resources:
-      requests:
-        storage: 1Mi
+    name: efs-sc
+  provisioner: efs.csi.aws.com
+  parameters:
+    provisioningMode: efs-ap
+    fileSystemId: <file-system-id>
```

Comment:
Is this parameterized with a $variable? Please confirm and also add comment for the user to supply this value